subprojects/kotlin-dsl/src/main/kotlin/org/gradle/kotlin/dsl/provider/KotlinScriptEvaluator.kt

```diff
@@ -254,7 +258,7 @@ class StandardKotlinScriptEvaluator(
 
         private
         val cacheKeyPrefix =
```

Comment:
:o: I would rename to `cacheKeySpecPrefix` for consistency

Reply:
Renamed.